index.js

```diff
@@ -384,7 +406,7 @@ var Purchases = /** @class */ (function () {
     };
     /**
      * Enable automatic collection of Apple Search Ad attribution. Disabled by default
-     * @param {Boolean} enabled Enable or not automatic apple search ads attribution collection
+     * @param {boolean} enabled Enable or not automatic apple search ads attribution collection
```

Comment:
i'm surprised we the have ability to control this?

Reply:
I'm not sure what you mean, do you mean the auto-generated `index.js`, the param or the type?

Reply:
that we can enable the attribution collection

Reply:
but i can do my own research on what that means 😅

Reply:
oh, I see. yeah, we can automatically get attribution from Apple Search Ads, under certain conditions: 
- the app isn't a Kids app
- the `AdServices` framework is included in the app
- on iOS < 14.5, if the user hasn't enabled Limit Ad Tracking or is otherwise restricted from tracking
- on iOS >= 14.5, this will only work if you also include `AppTrackingTransparency`, request tracking authorization and it gets granted